### ISSUE TITLE
Unit land/air status (colossus?)

### ISSUE THREAD

**Issue by N00byEdge** [Blizzard/s2client-api#99]
At the moment I can't find any information on what units occupy what layers. I see the bool is_flying in the sc2::Unit class, but there is no way (as far as I can tell) see that a marauder and a flying viking can hit a colossus without specifying that yourself. Am I missing something or is this intentional?

**Comment by KevinCalderone** [comment]
Yes it looks like we are missing this currently. I think colossus is the only special case?

**Comment by N00byEdge** [comment]
Yes but it would be nicer (and more general for the game) to have something like isAirUnit and isGroundUnit

**Comment by davechurchill** [comment]
I think what you want is more like: isAirTarget and isGroundTarget on the UnitType, with isFlying on the unit instance. That way you cover all cases

**Comment by KevinCalderone** [comment]
I agree isFlying is still useful. That way you can tell if it uses flying vs ground pathing.

**Comment by davechurchill** [comment]
isFlying is necessary, since some unit instances can switch, such as terran buildings

**Comment by N00byEdge** [comment]
I agree with what you're saying, just talking about adding some reliable way to see what weapons can target what.
